api/src/main/java/com/okta/sdk/oauth2/ErrorResponse.java

```diff
@@ -0,0 +1,32 @@
+package com.okta.sdk.oauth2;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ErrorResponse {
+
+    private String error;
+
+    @JsonProperty("error_description")
+    private String errorDescription;
+
+    public String getError() {
+        return error;
+    }
+
+    public void setError(String error) {
+        this.error = error;
+    }
+
+    public String getErrorDescription() {
+        return errorDescription;
+    }
+
+    public void setErrorDescription(String errorDescription) {
+        this.errorDescription = errorDescription;
+    }
+
+    @Override
+    public String toString() {
+        return "ErrorResponse [error=" + error + ", errorDescription=" + errorDescription + "]";
+    }
+}
```

Comment:
QUESTION: do we need this class or is there a general error class that can be used here?  can the error class for ssws be updated to be a general `AuthenticationErrorResponse`

Reply:
We dont necessarily need this class. But wanted to parse the error specific to OAuth token response more cleanly and present it. Its thus a good to have.

Reply:
If you are parsing OAuth errors then I suggest you use a more specific name, such as OAuthErrorResponse.

Reply:
This class is very similar to existing classes. It's contents should likely be converted into an `Error`, if you need a type specific version of that interface add it to the `impl` module

Reply:
This is very specific to OAuth2 response that is parsed from json body. It is not apt for this class to implement the `Error` interface internal to SDK, I feel.